src/backend/data_preprocessing.py

```diff
@@ -31,7 +31,7 @@ def __init__(self):
 
     # This function will read all the metadata of input images and put those metadata details in database.
     def process_metadata(self):
-        csv_file_path = os.getcwd()[:-6] + '/Data/HandInfo.csv'
+        csv_file_path = os.getcwd()[:-7] + '/Data/HandInfo.csv'
```

Comment:
Please post msg on slack so that everyone knows about this CSV file.

Reply:
Also, why have we changed it from 6 to 7? Folder name changed or something

Reply:
Yes folder changed from phase2 to backend so length of 7 now instead of 6